LICENSE

```diff
@@ -1,26 +1,22 @@
-LICENSE AGREEMENT For graphql.org software
+The MIT License (MIT)
 
-Facebook, Inc. (“Facebook”) owns all right, title and interest, including all
-intellectual property and other proprietary rights, in and to the graphql.org
-software. Subject to your compliance with these terms, you are hereby granted a
-non-exclusive, worldwide, royalty-free copyright license to (1) use and copy the
-graphql.org software; and (2) reproduce and distribute the graphql.org software
-as part of your own software (“Your Software”). Facebook reserves all rights not
-expressly granted to you in this license agreement.
+Copyright 2015 <Facebook, Inc.>
```

Comment:
Shouldn't this be `Copyright © 2020 The GraphQL Foundation` (or something similar)?